docs/src/ci-intro.md

```diff
@@ -126,7 +131,7 @@ jobs:
       with:
         distribution: 'temurin'
         java-version: '17'
-    - name: Build & Install
+    - name: Ensure browsers are installed
       run: mvn -B install -D skipTests --no-transfer-progress
     - name: Install Playwright
```

Comment:
```suggestion
    - name: Ensure browsers are installed
```

Reply:
good point, not sure what went wrong here. I've copied over the snippet from `ci.md` to keep them in sync. https://github.com/microsoft/playwright/pull/31988/commits/0be85a9ae65c679d35fe2e93f0b3576f758bd7b0